ngraph/frontend/paddlepaddle/src/op/squeeze.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "squeeze.hpp"
+#include <ngraph/opsets/opset6.hpp>
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+            namespace op
+            {
+                NamedOutputs squeeze(const NodeContext& node)
+                {
+                    auto data = node.get_ng_input("X");
+                    std::vector<int32_t> axes;
+                    if (node.has_attribute<std::vector<int32_t>>("axes"))
+                    {
+                        axes = node.get_attribute<std::vector<int32_t>>("axes");
+                    }
+
+                    auto axesNode =
+                        ngraph::opset6::Constant::create(ngraph::element::i32, {axes.size()}, axes);
+                    return node.default_single_output_mapping(
```

Comment:
Leave Squeeze with no axes input for the case if axes is empty

Reply:
@luo-cheng2021 Please handle the empty axes corner case

Reply:
Done